crates/extensions/collectd_ext/src/converter.rs

```diff
@@ -0,0 +1,26 @@
+use crate::batcher::MessageBatch;
+use crate::collectd::CollectdMessage;
+use batcher::BatchDriverOutput;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::Topic;
+use tracing::error;
```

Comment:
I realised, some crates use `tracing::error` and others use `log::error`.

Reply:
Fixed by https://github.com/thin-edge/thin-edge.io/pull/1817/commits/f9077a5226f70a16ffb7e0cb5937cecf6dce6f52